pontoon/base/views.py

```diff
@@ -356,30 +355,18 @@ def contributor(request, email, template='user.html'):
 def contributors(request, template='users.html'):
     """Top contributors view."""
     log.debug("Top contributors view.")
-
-    translations = (
-        Translation.objects
-        .exclude(user=None)
-        .exclude(string=F('entity__string'))
-        .exclude(string=F('entity__string_plural'))
-    )
-
-    users = (
-        User.objects
-        .filter(translation__in=translations).distinct()
-        .exclude(email__in=settings.EXCLUDE)
-        .annotate(translation_count=Count('translation'))
-        .exclude(translation_count=0)
-        .order_by('-translation_count')[:100]
-    )
-
-    for user in users:
-        user.translations = translations.filter(user=user)
+    try:
+        period = int(request.GET['period'])
+        start_date = (timezone.now() + relativedelta(months=-period))
+    except (KeyError, ValueError):
+        period = None
+        # When no given data, then we asume receive all data from 1970
+        start_date = timezone.make_aware(datetime.datetime.fromtimestamp(0))
```

Comment:
Since `start_date` is optional, might as well just set it to `None` here as it'll have the same effect.